app/domain/submission/functions.server.ts

```diff
@@ -79,11 +79,11 @@ const searchParams = (params: FilterParams): Parameters<typeof mongo.submissions
     ...(params.score
       ? {
           $or: [
-            params.score.includes("spam") ? { "score.avg": { $gte: 0, $lt: 25 } } : null,
-            params.score.includes("bad") ? { "score.avg": { $gte: 25, $lt: 50 } } : null,
-            params.score.includes("average") ? { "score.avg": { $gte: 50, $lt: 75 } } : null,
-            params.score.includes("good") ? { "score.avg": { $gte: 75, $lt: 100 } } : null,
-            params.score.includes("great") ? { "score.avg": { $gte: 100 } } : null,
+            params.score.includes("spam") ? { "score.avg": { $gte: 0, $lte: 24 } } : null,
```

Comment:
I think we should consolidate this business logic in a helper and use it in the searchSubmissionShowcase query as well.

```
export function scoreRange(score: "Stellar" | "Good" | "Average" | "Bad" | "Spam") {
  switch (score) {
    case "Stellar":
      return { $gte: 90 };
    case "Good":
      return { $gte: 70, $lt: 90 };
    case "Average":
      return { $gte: 45, $lt: 70 };
    case "Bad":
      return { $gte: 25, $lt: 45 };
    case "Spam":
      return { $lt: 25 };
  }
}
```

Reply:
I think I did this how you described?

Reply:
yes!